pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -60,19 +59,14 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	kubeClient, err := kubecli.GetKubevirtClientFromRESTConfig(mgr.GetConfig())
-	if err != nil {
-		log.Error(err, "Unable to get KubeVirt client")
-		panic("Controller cannot operate without KubeVirt")
-	}
 	tempClient, err := templatev1.NewForConfig(mgr.GetConfig())
 	if err != nil {
 		log.Error(err, "Unable to get OC client")
 		panic("Controller cannot operate without OC client")
 	}
 	client := mgr.GetClient()
 	finder := mappings.NewResourceMappingsFinder(client)
-	return &ReconcileVirtualMachineImport{client: client, scheme: mgr.GetScheme(), kubeClient: kubeClient, resourceMappingsFinder: finder, ocClient: tempClient}
+	return &ReconcileVirtualMachineImport{client: client, scheme: mgr.GetScheme(), resourceMappingsFinder: finder, ocClient: tempClient}
```

Comment:
maybe we could get rid of `ocClient` as well?

Reply:
Yes, this PR removes only kubevirt client. I will send another one with oclient

Reply:
I meant the last parameter

```
ocClient: tempClient
```

Reply:
Yes, I know, but it would mean remove it from all places same as kubevirt client. And it's quite lot changes, so I wanted to do this patch smaller.